File: batch/newjoy_notification/PlaceOfActivity.py
from enum import Enum


class PlaceOfActivity(Enum):
    NEWJOY_DISCORD = 'NEWJOY_DISCORD'
    DISCORD = 'DISCORD'
    MINE = 'MINE'
    YOTO = 'YOTO'
    ZOOM = 'ZOOM'
    OTHER = 'OTHER'


class PlaceOfActivityTrans(Enum):
    NEWJOY_DISCORD = '新歓ディスコード'
    DISCORD = 'ディスコード'
    MINE = '峰キャンパス'
    YOTO = '陽東キャンパス'
    ZOOM = 'Zoom'
    OTHER = 'その他'


def placeOfActivityTrans(key: str) -> str:
    if key == PlaceOfActivityTrans.NEWJOY_DISCORD.name:
        return PlaceOfActivityTrans.NEWJOY_DISCORD.value

    if key == PlaceOfActivityTrans.DISCORD.name:
        return PlaceOfActivityTrans.DISCORD.value

    if key == PlaceOfActivityTrans.MINE.name:
        return PlaceOfActivityTrans.MINE.value

    if key == PlaceOfActivityTrans.YOTO.name:
        return PlaceOfActivityTrans.YOTO.value

    if key == PlaceOfActivityTrans.ZOOM.name:
        return PlaceOfActivityTrans.ZOOM.value

    if key == PlaceOfActivityTrans.OTHER.name:
        return PlaceOfActivityTrans.OTHER.value

    return key

File: api/app/Enum/PlaceOfActivity.php
<?php

namespace App\Enum;

/**
 * 活動場所
 */
class PlaceOfActivity
{
    /**
     * 峰キャンパス
     */
    const MINE = 'MINE';

    /**
     * 陽東キャンパス
     */
    const YOTO = 'YOTO';

    /**
     * 峰キャンパス, 陽東キャンパス
     */
    const MINE_AND_YOTO = 'MINE_AND_YOTO';

    /**
     * 新歓Discord
     */
    const NEWJOY_DISCORD = 'NEWJOY_DISCORD';

    /**
     * Zoom
     */
    const ZOOM = 'ZOOM';

    /**
     * Discord
     */
    const DISCORD = 'DISCORD';

    /**
     * その他
     */
    const OTHER = 'OTHER';

    /**
     * 新歓の活動場所
     *
     * @return PlaceOfActivity[]
     */
    public static function toArrayForCircleNewJoy(): array
    {
        return [
            PlaceOfActivity::MINE,
            PlaceOfActivity::YOTO,
            PlaceOfActivity::NEWJOY_DISCORD,
            PlaceOfActivity::DISCORD,
            PlaceOfActivity::ZOOM,
            PlaceOfActivity::OTHER
        ];
    }
}
